components/ws-manager/cmd/run.go

```diff
@@ -118,6 +119,14 @@ var runCmd = &cobra.Command{
 		}
 		defer mgmt.Close()
 
+		err = watch.File(context.Background(), cfgFile, func() {
+			cfg := getConfig()
+			mgmt.Config = cfg.Manager
+		})
+		if err != nil {
+			log.WithError(err).Fatal("cannot start watch of configuration file")
```

Comment:
```suggestion
			log.WithError(err).WithField("cfgFile", cfgFile).Fatal("cannot start watch of configuration file")
```
to know where it is looking for config